venza.py

```diff
@@ -28,7 +28,10 @@ def __str__(self):
         return self.fmt("{name} - {place}")
 
     def __repr__(self):
-        return self.fmt("<Meal('{name}', place='{place}')>")
+        return self.fmt("<Meal('{name}',"
+                        " place='{place}'"
+                        " price='{price}'"                        
```

Comment:
@tWido can you please remove the quotiation marks (`'`) from this line? Price is no longer a string